src/kyc.ts

```diff
@@ -23,11 +23,21 @@ export const kycStatusSchema = z.nativeEnum(KycStatus, {
 // When adding new schemas be sure to also update kycSchemasSchema
 export enum KycSchema {
   PersonalDataAndDocuments = 'PersonalDataAndDocuments',
+  PersonalDataAndDocumentsDetailed = 'PersonalDataAndDocumentsDetailed',
 }
 export const kycSchemaSchema = z.nativeEnum(KycSchema, {
   description: 'kycSchemaSchema',
 })
 
+export enum KycDocumentType {
```

Comment:
Please follow the naming convention used in the specification itself; this should be [IdentificationDocumentType](https://github.com/fiatconnect/specification/blob/main/fiatconnect-api.md#93121-identificationdocumenttype).

Reply:
Sure, changed.